protocol/0028-GOVE-governance.md

```diff
@@ -317,9 +316,8 @@ The proposal specifies:
 - `fraction_of_balance`: the maximum fraction of the source account's balance to transfer as a decimal (i.e. 0.1 = 10% of the balance)
 - `destination_type` specifies the account type to transfer to (reward pool, party, network insurance pool, market insurance pool)
 - `destination` specifies the account to transfer to, depending on the account type:
-  - reward pool: the reward scheme ID
+  - network treasury: leave blank (only one per asset)
   - party: the party's public key
-  - network insurance pool: leave blank (there's only one per asset)
```

Comment:
Since the insurance pool is per market, do we need to provide market ID for the insurance pool?